independent-publisher-connectors/Microsoft 365 Service Health/apiDefinition.swagger.json

```diff
@@ -0,0 +1,350 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft 365 Service Health",
+    "description": "Provides access to the Service Announcements and Service Health feeds that exist on the Microsoft Graph (BETA).  Please read the setup instructions for configuring an app registration for this connector to use for Graph access.  For more on the Graph Service Announcements api please see https://docs.microsoft.com/en-us/graph/api/resources/service-communications-api-overview?view=graph-rest-beta",
+    "version": "1.0",
+    "contact": {
+      "name": "Nick Brown",
+      "url": "https://nbdev.uk"
+    }
+  },
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://nbdev.uk"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "Microsoft;Microsoft 365;Service Status;Service Health;Service Announcements;Microsoft Graph;BETA"
```

Comment:
@techienickb These category values are incorrect. Please provide the predefined category values only. And maximum two categories are allowed.

Documentation - [Product or end service metadata](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#step-4b-product-or-end-service-metadata)

Reply:
Adjusted as requested, I've also switched from beta graph to v1.0 as the api is now released